<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_1">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>属性</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser_2"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionread"/>
    <addaction name="actionsave"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关键点</string>
    </property>
   </widget>
   <widget class="QMenu" name="filters">
    <property name="title">
     <string>滤波</string>
    </property>
    <widget class="QMenu" name="menu_11">
     <property name="title">
      <string>采样</string>
     </property>
     <addaction name="actionvoxel"/>
     <addaction name="actionuniform"/>
     <addaction name="actionrandom"/>
    </widget>
    <widget class="QMenu" name="menu_12">
     <property name="title">
      <string>平滑</string>
     </property>
     <addaction name="actionmiddle"/>
     <addaction name="actionmean"/>
    </widget>
    <widget class="QMenu" name="menu_13">
     <property name="title">
      <string>去噪</string>
     </property>
     <addaction name="actionradius"/>
     <addaction name="actionsor"/>
    </widget>
    <addaction name="menu_11"/>
    <addaction name="menu_12"/>
    <addaction name="menu_13"/>
   </widget>
   <widget class="QMenu" name="registration">
    <property name="title">
     <string>配准</string>
    </property>
    <widget class="QMenu" name="menuGlobal">
     <property name="title">
      <string>Global</string>
     </property>
     <addaction name="actionFGR"/>
    </widget>
    <widget class="QMenu" name="menu_14">
     <property name="title">
      <string>点云网络</string>
     </property>
     <addaction name="actionDCP"/>
    </widget>
    <widget class="QMenu" name="menuICP">
     <property name="title">
      <string>ICP</string>
     </property>
     <addaction name="actionicp2point"/>
     <addaction name="actionicp2Plane"/>
    </widget>
    <addaction name="menuICP"/>
    <addaction name="menuGlobal"/>
    <addaction name="menu_14"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>分割</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>识别</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>重建</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="filters"/>
   <addaction name="registration"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionread"/>
   <addaction name="actionsave"/>
   <addaction name="actionquit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionviewup"/>
   <addaction name="actionviewerdown"/>
   <addaction name="actionviewfront"/>
   <addaction name="actionviewback"/>
   <addaction name="actionviewleft"/>
   <addaction name="actionviewright"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionread">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_read.png</normaloff>:/Icons/Icons/ic_read.png</iconset>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_save.png</normaloff>:/Icons/Icons/ic_save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_quit.png</normaloff>:/Icons/Icons/ic_quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionviewup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_up.png</normaloff>:/Icons/Icons/ic_up.png</iconset>
   </property>
   <property name="text">
    <string>viewup</string>
   </property>
  </action>
  <action name="actionviewerdown">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_down.png</normaloff>:/Icons/Icons/ic_down.png</iconset>
   </property>
   <property name="text">
    <string>viewerdown</string>
   </property>
  </action>
  <action name="actionviewfront">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_front.png</normaloff>:/Icons/Icons/ic_front.png</iconset>
   </property>
   <property name="text">
    <string>viewfront</string>
   </property>
  </action>
  <action name="actionviewback">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_back.png</normaloff>:/Icons/Icons/ic_back.png</iconset>
   </property>
   <property name="text">
    <string>viewback</string>
   </property>
  </action>
  <action name="actionviewleft">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_left.png</normaloff>:/Icons/Icons/ic_left.png</iconset>
   </property>
   <property name="text">
    <string>viewleft</string>
   </property>
  </action>
  <action name="actionviewright">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/ic_right.png</normaloff>:/Icons/Icons/ic_right.png</iconset>
   </property>
   <property name="text">
    <string>viewright</string>
   </property>
  </action>
  <action name="actionvoxel">
   <property name="text">
    <string>体素</string>
   </property>
  </action>
  <action name="actionuniform">
   <property name="text">
    <string>均匀</string>
   </property>
  </action>
  <action name="actionrandom">
   <property name="text">
    <string>随机</string>
   </property>
  </action>
  <action name="actionmiddle">
   <property name="text">
    <string>中值</string>
   </property>
  </action>
  <action name="actionmean">
   <property name="text">
    <string>均值</string>
   </property>
  </action>
  <action name="actionradius">
   <property name="text">
    <string>半径</string>
   </property>
  </action>
  <action name="actionsor">
   <property name="text">
    <string>统计</string>
   </property>
  </action>
  <action name="actionFGR">
   <property name="text">
    <string>FGR</string>
   </property>
  </action>
  <action name="actionDCP">
   <property name="text">
    <string>DCP</string>
   </property>
  </action>
  <action name="actionicp2point">
   <property name="text">
    <string>icp2Point</string>
   </property>
  </action>
  <action name="actionicp2Plane">
   <property name="text">
    <string>icp2Plane</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
